src/wcs1_0/src/test/java/org/geoserver/wcs/GetCoverageTest.java

```diff
@@ -497,7 +499,33 @@ public void testTimeFirstKVP() throws Exception {
         
         checkPixelValue(response, 10, 10, 18.2659999176394);
     }
-    
+
+    @Test
+    public void testTimeTooMany() throws Exception {
+        GeoServer gs = getGeoServer();
+        WCSInfo wcs = gs.getService(WCSInfo.class);
+        wcs.setMaxRequestedDimensionValues(2);
+        gs.save(wcs);
+        try {
+            String queryString =
+                    "request=getcoverage&service=wcs&version=1.0.0&format=image/geotiff"
+                            + "&bbox=0.237,40.562,14.593,44.558&crs=EPSG:4326&width=25&height=25&time=2008-10-31/2008-11-31/PT1H"
+                            + "&coverage="
+                            + getLayerId(WATTEMP);
+            MockHttpServletResponse response = getAsServletResponse("wcs?" + queryString);
+            assertEquals("application/vnd.ogc.se_xml", response.getContentType());
+            Document dom = dom(response, true);
+            // print(dom);
```

Comment:
commented out line.

Reply:
In test code it's normal to have those, there are hundreds:

````
> git grep "// print" | wc -l
702
> git grep "//print" | wc -l
80
````

Rationale is easy, we want to avoid printing out stuff in a normal build, but having it handy for debugging tests is useful.

Reply:
Yep. Indeed I was thinking about something related to quick debug. Sorry for the noise :)